CHANGES.md

```diff
@@ -1,5 +1,36 @@
 # RxJava Releases #
 
+### Version 1.0.0 - August 5th 2015 ####
+
+Initial stable release!
+
+In order to provide a library that no project using RxJava would hesitate to depend on, the decision
+was made to remove the APIs which were not absolutely fundamental to all apps. That is what's
+contained in this release.
+
+Functionality which was previously part of this library is being explored in separate, modular
+libraries:
+
+ * `AppObservable`: https://github.com/trello/RxLifecycle
```

Comment:
This actually replaces `LifecycleObservable`.

Nothing replaces `AppObservable`. I think it was unnecessary anyways. I suspect it was always the fact that the binding operators weren't checking `isUnsubscribed()`.